dkim.js

```diff
@@ -449,7 +449,7 @@ class DKIMVerifyStream extends Stream {
             buf = this.buffer.pop(buf);
         }
 
-        const callback = function (err, result) {
+        const callback = (err, result) => {
```

Comment:
```suggestion
        function callback (err, result) {
```

Reply:
We've been replacing those function assignments with function declarations in most cases. It's generally for the best since `const` is not hoisted and function declarations are.